### DISCUSSION TITLE
Container Tag warning gets eaten up for no reason

### DISCUSSION THREAD

**Discussion by ubertpendragon** [FakeFishGames/Barotrauma#14332] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

Container Tag warning window displays `4 more... ` with enough room for 50 more

![image](https://github.com/user-attachments/assets/2f7f7547-4264-478d-ad91-9859920adc73)

### Reproduction steps

create new sub w cabinets and save

### Bug prevalence

Just once

### Single player or multiplayer?

Single player

### Version

v1.5.9.1 (Summer Update Hotfix 2)

### Which operating system did you encounter this bug on?

Windows

**Comment by Regalis11** [comment]
This is not a bug, the warning only includes the first 3 tags.

**Reply by ubertpendragon** [reply]
But why, there's room...

It's annoying to save a bunch of times to reveal the missing ones 3 by 3

**Reply by Regalis11** [reply]
Mainly just for technical reasons (or, our laziness to change how these warnings are handled :D) - the method that generates these warnings about the submarine has no knowledge of where the warnings will be used or how much space there will be, and it truncates this specific message to avoid making the warning excessively long. 

This warning wasn't really intended to be used for figuring out which tags are missing anyway (I guess you would need to take a screenshot or something if you want to use it and remember which 15 tags were missing). It's supposed to just notify the user that there's some issues with the tags, and they're intended to be fixed in the container tag menu which lets you see what's missing a lot more easily.
